x-pack/platform/plugins/shared/automatic_import_v2/server/routes/utils.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  KibanaResponseFactory,
+  CustomHttpResponseOptions,
+  HttpResponsePayload,
+  ResponseError,
+} from '@kbn/core/server';
+
+const statusToErrorMessage = (statusCode: number) => {
```

Comment:
This must already exist in return messages, is it just that you are returning only the status code in the response or? A bit unsure why we have to manually translate these